frontend/src/components/Button.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  export let title;
+  export let fontSize = '1em';
+</script>
+
+<button on:click styling="font-size: {fontSize}">{title}</button>
```

Comment:
shouldn't the attribut's name be `style`?
or is this a sveltism I don't know about?

Reply:
🤷‍♂️... but it is not used